sdk/face/azure-ai-vision-face/assets.json

```diff
@@ -0,0 +1,6 @@
+{
+  "AssetsRepo" : "Azure/azure-sdk-assets",
+  "AssetsRepoPrefixPath" : "java",
+  "TagPrefix" : "java/face/azure-ai-vision-face",
+  "Tag" : ""
```

Comment:
We should record the tests and push the recordings to the assets repo. This will update the tag.

Reply:
Got it. I am working on it.